lib/ansible/modules/windows/win_get_url.ps1

```diff
@@ -19,120 +20,186 @@
 # WANT_JSON
 # POWERSHELL_COMMON
 
+
+Function CheckModified-File($url, $dest, $headers, $credentials, $timeout, $use_proxy, $proxy) {
```

Comment:
So what should `$proxy` than become if `$use_proxy` is `$true` (and there is no `$proxy_url` set) ?